### ISSUE TITLE
Segfault in insert routing policy

### ISSUE THREAD

**Issue by maxwedwards** [lncapital/torq#138]
```
{"level":"info","time":"2022-10-04T13:20:04Z","message":"Stopping subscriptions"}
{"level":"info","time":"2022-10-04T13:20:04Z","message":"All subscriptions stopped"}
{"level":"info","time":"2022-10-04T13:20:04Z","message":"Restarting subscriptions"}
[GIN] 2022/10/04 - 13:20:04 | 200 |   21.387759ms |  192.168.100.76 | PUT      "/api/settings/local-nodes/2"
{"level":"info","time":"2022-10-04T13:20:04Z","message":"Subscribing to LND for node id: 2"}
[GIN] 2022/10/04 - 13:20:04 | 200 |    4.431746ms |  192.168.100.76 | GET      "/api/settings/local-nodes"
[GIN] 2022/10/04 - 13:20:04 | 200 |    5.239919ms |  192.168.100.76 | GET      "/api/settings/local-nodes/2"
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x48 pc=0xa98974]

goroutine 180 [running]:
github.com/lncapital/torq/pkg/lnd.insertRoutingPolicy(0x40004f6420, {0x40005a8880?, 0x2?, 0x0?}, 0x0, 0x40003f78f0)
 /app/pkg/lnd/channelgraph.go:178 +0x2a4
github.com/lncapital/torq/pkg/lnd.ImportRoutingPolicies({0x116c2c0, 0x400002a5e0}, 0x40004f6420, {0x40005a8880, 0x2, 0x2})
 /app/pkg/lnd/channelinfo.go:131 +0x394
github.com/lncapital/torq/cmd/torq/internal/subscribe.Start({0x1157ba0, 0x4000660fc0}, 0x40001e8000, 0x40004f6420, 0x2)
 /app/cmd/torq/internal/subscribe/server.go:86 +0x298
main.main.func1.2.1({0x2, {0x4000041260, 0x13}, {0x4000cfb500, 0x32e, 0x32e}, {0x4000730700, 0xd9, 0xd9}})
 /app/cmd/torq/torq.go:200 +0x120
created by main.main.func1.2
 /app/cmd/torq/torq.go:183 +0x1fc
 ```

**Comment by 2Fast2BCn** [comment]
Does this also fix issue #99 ?

**Comment by ohenrik** [comment]
This has been fixed in after version 0.8.16 by 4cee697.
